packages/suite/src/actions/wallet/__fixtures__/blockchainActions.ts

```diff
@@ -1,6 +1,6 @@
-import { NOTIFICATION } from '@suite-actions/constants';
 import { analyzeTransactionsFixtures as analyzeTransactions } from '@suite-common/wallet-utils';
 import { blockchainActions, transactionsActions, accountsActions } from '@suite-common/wallet-core';
+import { addEventThunk } from '@suite-common/in-app-notifications';
```

Comment:
`addToastThunk` sounds okay. However, `addEventThunk` sounds confusing to me.

What about `addToastNotificationThunk` and `AddEventNotificationThunk`?

What's the difference between `toast` and `event`? Only `seen` attribute? Do we need such a complication?

Reply:
I kept it in the end it's used in more than one place. But it's now namespaced in `notificationsActions.addToast/Event` so it makes better sense now.